### DISCUSSION TITLE
Lock file error.

### DISCUSSION THREAD

**Discussion by purushothamanmunusamy** [gumyr/build123d#696] [Q&A]
I am new to python and build123d. please help me fix the error. 

```
Lock file C:\Users\info\.ocpvscode.lock already exists
Lock file C:\Users\info\.ocpvscode.lock is stale
show_all: Unable to open config file C:\Users\info\.ocpvscode.
```
and my code is 

```
from build123d import *
from ocp_vscode import *

with BuildPart() as example:
    Cylinder(radius=10, height=3)

show_all()
```

I am using recent version of pycharm IDE.

if you need any other information. please let me know.

**Comment by gumyr** [comment]
This is an ocp_vscode question which should be asked on that tool's GitHub page: https://github.com/bernhard-42/vscode-ocp-cad-viewer
